Exiled.API/Features/Respawn.cs

```diff
@@ -23,6 +23,9 @@ namespace Exiled.API.Features
     /// </summary>
     public static class Respawn
     {
+        private const string MtfChopperGameObjectName = "Chopper";
+        private const string ChaosVanGameObjectName = "CIVanArrive";
```

Comment:
I don't see how moving immutable strings to a constant variable is any different to just hard-referencing the exact same string.

Reply:
facilitate maintenance, avoid searching everywhere in the class for names that might change. And allows you to use the same name everywhere and edit only in one place.

Reply:
> facilitate maintenance, avoid searching everywhere in the class for names that might change. And allows you to use the same name everywhere and edit only in one place.

Since these values are only ever used once, I don't see the need for it, but I understand how it's more efficient for maintainability.

Reply:
You only need to look up in the class if the GameObject name change and not in the all class. 
And that is use full to avoid magical value.